src/main/java/org/mule/tools/maven/repository/ArtifactInstaller.java

```diff
@@ -95,4 +160,24 @@ protected static File getFormattedOutputDirectory(File outputDirectory, Artifact
 
     return new File(outputDirectory, sb.toString());
   }
+
+  public void downloadExcludedDependencyToLocalRepository(Artifact artifact) throws MojoExecutionException {
+    CollectRequest collectRequest = new CollectRequest();
+    collectRequest.setRoot(new org.eclipse.aether.graph.Dependency(RepositoryUtils.toArtifact(artifact), null));
```

Comment:
No, valid point. While coding I was using both Dependency classes from Aether and Maven API, but now it can be refactored.